website/config.toml

```diff
@@ -113,10 +113,8 @@ algolia_docsearch = false
 
 # User interface configuration
 [params.ui]
-# Enable to show the side bar menu in its compact state.
-sidebar_menu_compact = true
```

Comment:
I think we need the `sidebar_menu_compat = true` for the SDK docs.

![Screenshot from 2020-11-02 15-22-36](https://user-images.githubusercontent.com/1028657/97915312-485a4800-1d1f-11eb-99f3-b6ad30d082be.png)